src/Sylius/Component/Core/composer.json

```
@@ -52,18 +52,18 @@
         "sylius/taxation": "^2.0",
         "sylius/taxonomy": "^2.0",
         "sylius/user": "^2.0",
-        "symfony/http-client": "^5.4.21 || ^6.0",
-        "symfony/http-foundation": "^5.4.21 || ^6.0",
+        "symfony/http-client": "^6.3.2",
+        "symfony/http-foundation": "^6.3.4",
         "symfony/mime": "^5.4.21 || ^6.0",
```

```suggestion
        "symfony/mime": "^6.3.0",
```
IMO, we can bump symfony/meme too.

---

You can call yourself an eagle eye from now 😅